ksqldb-rest-app/src/main/java/io/confluent/ksql/api/server/ServerVerticle.java

```diff
@@ -116,6 +118,9 @@ int actualPort() {
   private Router setupRouter() {
     final Router router = Router.router(vertx);
 
+    router.route(HttpMethod.GET, "/chc/ready").handler(ServerVerticle::chcHandler);
```

Comment:
Can we add a comment explaining the importance of handling these before the server state handler (and auth handler)? It's great that you've added tests for this but I think a comment would be nice for anyone who happens to be glancing at the code.

Reply:
>> Are we confident that users don't rely on this functionality?

I did a search in all confluentinc repos and /chc seems to be the only user of this. In terms of community users this was never a documented feature, afaik.

Reply:
OK. Let's prepare a blurb for the changelog/release notes with a summary of implications from the migration from Jetty to Vert.x, including that some Jetty-specific plugins/features may not be supported in full? There's almost certainly a more artful way to word this but I think it'd be good to call out.

The auto-generated changelog will pick it up if the commit message (the one that actually gets merged to GitHub) includes `BREAKING CHANGE: <text here>`. Or we can manually insert it into the changelog/release notes come release time.

Reply:
Hmm, AIUI the fact that we were previously using Jetty was an implementation detail, and we've never had any documented features reliant on it.